locales/en.default.schema.json

```diff
@@ -509,6 +509,10 @@
         },
         "show_vendor": {
           "label": "Show vendor"
+        },
+        "show_rating": {
+          "label": "Show product rating",
+          "info": "To display a rating, add a product ratings app. [Learn more](https:\/\/help.shopify.com\/en\/manual\/products\/product-reviews\/installation)"
```

Comment:
Would apply to each use: remove "s" from product rating**s** app

Final-final-final:
`To display a rating, add a product rating app. Learn more [offsite icon]`